### DISCUSSION TITLE
Comment insérer des couches geojson dans itowns?

### DISCUSSION THREAD

**Discussion by LHOTTELIERThomas** [iTowns/itowns#2065] [Q&A]
Bonjour, 

Étant débutant dans le domaine du code, j'aimerais savoir simplement, comment ajouter des couches geojson dans itowns ?

Et s'il existe des tutoriels pour apprendre à l'utiliser ?

Merci d'avance pour vos réponses !

**Comment by jailln** [comment]
Bonjour,
Il y a plusieurs tutoriels dans l'onglet documentation du [site web d'itowns](https://www.itowns-project.org/) qui permettent d'apprendre à utiliser itowns.
Concernant l'ajout de geojson, vous pouvez vous inspirer d'un des deux exemples suivants:
- [GeoJSON to raster](https://www.itowns-project.org/itowns/examples/index.html#source_file_geojson_raster): charge un geojson, le convertit en raster et l'affiche.
- [GeoJSON to 3D Object](https://www.itowns-project.org/itowns/examples/index.html#source_file_geojson_3d): charge un geojson, le convertit en objet 3D et l'affiche.

**Reply by LHOTTELIERThomas** [reply]
Merci pour vos réponses!

Après avoir effectué plusieurs tests, j'ai constaté que mes données geojson ne sont pas prises en charge. Est-ce que vous pourriez éventuellement me fournir un code avec des données geojson chargées en local ? Les données en question ont été extraites de QGIS.

Merci par avance pour votre aide!

**Reply by mgermerie** [reply]
Bonjour,

Pourriez-vous éventuellement nous fournir le fichier GeoJSON en question ? Il est compliqué de produire un code générique chargeant nimporte quelle donnée GeoJSON, étant donné que les paramètres de la source iTowns vont dépendre des données, et que le style du calque qui supporte l'affichage de ces données va également dépendre des données (et de la façon dont elles doivent être représentées).

J'en profite également pour vous orienter vers [ce tutoriel](http://www.itowns-project.org/itowns/docs/#tutorials/Vector-data-on-ground) qui explique comment afficher le contenu de deux GeoJSON avec iTowns.
